lib/ApiResource.php

```diff
@@ -103,7 +79,7 @@ public static function baseUrl()
      */
     public static function classUrl()
     {
-        $base = static::className();
+        $base = str_replace('.', '/', static::OBJECT_NAME);
```

Comment:
we likely should add a comment about that one to be clearer/more discoverable in the future